src/lib/Scenes/Map/GlobalMap.tsx

```diff
@@ -226,6 +227,16 @@ export class GlobalMap extends React.Component<Props, State> {
     this.setState({ activeIndex, activePin: null, activeShows: [] }, () => this.emitFilteredBucketResults())
   }
 
+  resetZoomAndCamera = () => {
+    this.map.setCamera({
```

Comment:
I've had some issues with the `setCamera` call in the past but I was only setting `zoom` and `duration`, we should test it out a bit when the pitch and heading are already at 0.